Detectors/GlobalTrackingWorkflow/src/globalfwd-matcher-workflow.cxx

```diff
@@ -35,6 +37,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
     {"disable-mc", o2::framework::VariantType::Bool, false, {"disable MC propagation even if available"}},
     {"disable-root-input", o2::framework::VariantType::Bool, false, {"disable root-files input reader"}},
     {"disable-root-output", o2::framework::VariantType::Bool, false, {"do not write output root files"}},
+    {"enable-match-output", o2::framework::VariantType::Bool, false, {"stores mftmch matching info on mftmchmatches.root"}},
```

Comment:
cannot this be handled by the `disable-root-output` ?

Reply:
This is a special case to store a separate file with track matching associations (not full tracks) for debug purposes. In general this file is not necessary as  globalfwdtracks.root have the full information and is produced by the default workflow. This option exists only for debugging and  matching performance studies.